Thanks to TGL <degrenier@easyconnect.fr> in http://bugs.gentoo.org/131837

already in upstream cvs:
http://webcvs.freedesktop.org/mesa/Mesa-oldtree/src-glut/Makefile.am?r1=1.19&r2=1.20

--- Mesa-5.0/src-glut/Makefile.am.orig	2006-04-30 15:39:32.000000000 +0200
+++ Mesa-5.0/src-glut/Makefile.am	2006-04-30 15:39:47.000000000 +0200
@@ -9,7 +9,6 @@
 EXTRA_PROGRAMS = capturexfont
 
 libglut_la_LDFLAGS = -version-info @LIBGLUT_VERSION_INFO@ -L../@GLU_DIR@
-libglut_la_LIBADD = -lGLU
 libglut_la_SOURCES = \
 	glutwin32.h \
 	win32_glx.h \
@@ -67,7 +66,7 @@
 	layerutil.h \
 	stroke.h
 
-libglut_la_LIBADD = $(X_LIBS) $(X_PRE_LIBS) $(X_LIBADD)
+libglut_la_LIBADD = -lGLU $(X_LIBS) $(X_PRE_LIBS) $(X_LIBADD)
 capturexfont_LDFLAGS = $(X_LIBS) $(X_PRE_LIBS) $(X_LIBADD)
 
 EXTRA_DIST = descript.mms
